src/components/AccordionSection/index.js

```diff
@@ -142,6 +152,7 @@ class AccordionItem extends Component {
                             ariaExpanded={isExpanded}
                             ref={this.buttonRef}
                             icon={<RightArrow isExpanded={isExpanded} disabled={disabled} />}
+                            variant={variant}
```

Comment:
Remember, changes to component API needs to be discussed. In this case, Accordion does not have a variant, for now, the only thing that changes is the right arrow button color.